### ISSUE TITLE
Unable to run migrations (migrations table always locked)

### ISSUE THREAD

**Issue by majidmvulle** [uptrace/bun#64]
Attempting to run migrations on a brand new database throws the error below

```
[bun]  18:24:14.317   INSERT                  100µs  INSERT INTO bun_migration_locks AS l (`id`, `table_name`) VALUES (DEFAULT, 'bun_migrations') 	  *mysql.MySQLError: Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'AS l (`id`, `table_name`) VALUES (DEFAULT, 'bun_migrations')' at line 1
2021/07/07 18:24:14 bun: migrations table is already locked (Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'AS l (`id`, `table_name`) VALUES (DEFAULT, 'bun_migrations')' at line 1)
```

My go.mod file is as below:

```
module mymodule

go 1.16

require (
	github.com/PuerkitoBio/goquery v1.7.0
	github.com/go-sql-driver/mysql v1.6.0
	github.com/spf13/cobra v1.2.1
	github.com/spf13/viper v1.8.1
	github.com/uptrace/bun v0.2.14
	github.com/uptrace/bun/dialect/mysqldialect v0.2.14
	github.com/uptrace/bun/extra/bundebug v0.2.14
	github.com/urfave/cli/v2 v2.3.0
	go4.org v0.0.0-20201209231011-d4a079459e60
)
```

**Comment by majidmvulle** [comment]
> Which MySQL version you are using? MySQL 5.7 is minimal supported version.

I have tried it on version MySQL 5.7.32 on Big Sur as well as MariaDB 10.6.3 on Ubuntu 20. Same issue. Please see the DB versions in the screenshots.

![Screen Shot 2021-07-08 at 09 35 39](https://user-images.githubusercontent.com/1475098/124867986-24e34d80-dfd0-11eb-9869-a141e4227d59.png)
![Screen Shot 2021-07-08 at 09 34 07](https://user-images.githubusercontent.com/1475098/124867990-26147a80-dfd0-11eb-8f1b-6caa33b3fcde.png)

**Comment by vmihailenco** [comment]
https://github.com/uptrace/bun/pull/66 should fix this, but we are in the middle of refactoring so it may take day or two to make a release.

Meanwhile you can either try to upgrade to MySQL 8 (which does not have this problem) or use the unreleased (master) version.
-

**Comment by vmihailenco** [comment]
Should be fixed in v0.3.0
